docs/pages/cairo/hello-dojo.md

```diff
@@ -238,13 +233,13 @@ katana --disable-fee
 Success! [Katana](/toolchain/katana/overview.md) should now be running locally on your machine. Now, let's deploy! In your primary terminal, execute:
 
 ```bash
-sozo migrate
+sozo migrate apply
 ```
 
-This command will deploy the artifact to [Katana](/toolchain/katana/overview.md). You should see terminal output similar to this:
+This command will deploy the artifact to `Katana`. You should see terminal output similar to this:
```

Comment:
We should keep the link I guess, to ensure people can navigate to Katana while reading this. 👍